admin-portal/src/components/navbar/navbar.tsx

```diff
@@ -1,10 +1,66 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
+
+import { Hooks, useAuthContext } from "@asgardeo/auth-react";
 
 import "./navbar.css";
+import { useLocation } from "react-router-dom";
 
 interface NavBarProps {}
 
 export function NavBar(params: NavBarProps) {
+  const { state, signIn, signOut, on, trySignInSilently } = useAuthContext();
+
+  // const search = useLocation().search;
+  // const stateParam = new URLSearchParams(search).get("state");
+  // const errorDescParam = new URLSearchParams(search).get("error_description");
+
+  // useEffect(() => {
+  //   if (stateParam && errorDescParam) {
+  //     if (errorDescParam === "End User denied the logout request") {
+  //       setHasLogoutFailureError(true);
+  //     }
+  //   }
+  // }, [stateParam, errorDescParam]);
+
+  /**
+   * handles the error occurs when the logout consent page is enabled
+   * and the user clicks 'NO' at the logout consent page
+   */
+  useEffect(() => {
+    on(Hooks.SignOut, () => {
+      //setHasLogoutFailureError(false);
+    });
+  }, [on]);
+
+  useEffect(() => {
+    if (!state?.isAuthenticated) {
+      return;
+    }
+    // (async (): Promise<void> => {
+    //   const basicUserInfo = await getBasicUserInfo();
+    //   const idToken = await getIDToken();
+    //   const decodedIDToken = await getDecodedIDToken();
+
+    //   const derivedState = {
+    //     authenticateResponse: basicUserInfo,
+    //     idToken: idToken?.split("."),
+    //     decodedIdTokenHeader: JSON.parse(atob(idToken?.split(".")[0])),
+    //     decodedIDTokenPayload: decodedIDToken
+    //   };
+
+    //   setDerivedAuthenticationState(derivedState);
+    // })();
+  }, [state?.isAuthenticated]);
+
+  const handleLogin = () => {
+    //setHasLogoutFailureError(false);
+    signIn(); //.catch(() => setHasAuthenticationErrors(true));
```

Comment:
```suggestion
  const handleLogin = () => {
    signIn();
```

Reply:
Let's keep the commented code as we need these to handle errors